### PR TITLE
Allow set of GALAXY_MINIMUM_PASSWORD_LENGTH for AUTH_PASSWORD_VALIDATORS

### PR DESCRIPTION
**Pull request by rochacbruno**
Issue: AAH-1531

Allow set of GALAXY_MINIMUM_PASSWORD_LENGTH for AUTH_PASSWORD_VALIDATORS

https://deploy-preview-1253--galaxyng.netlify.app/config/user_access/

# Description 🛠
_Add PR description here_

# Reviewer Checklists  👀
Developer reviewer:
- [ ] Code looks sound, good architectural decisions, no [code smells](https://www.codegrip.tech/productivity/everything-you-need-to-know-about-code-smells/)
- [ ] There is a Jira issue associated (note that "No-Issue" should be rarely used)
- [ ] Tests are included in `galaxy_ng/tests/integration` or `galaxy_ng/tests/functional`, and they fully cover necessary test scenarios… or tests not needed

QE reviewer ([exceptions](https://docs.engineering.redhat.com/display/AUTOHUB/Other+Team+Processes#OtherTeamProcesses-galaxy_ngrepo)):
- [ ] Tests are included in `galaxy_ng/tests/integration` or `galaxy_ng/tests/functional`, and they fully cover necessary test scenarios… or tests not needed
- [ ] PR meets applicable Acceptance Criteria for associated Jira issue

Note: when merging, include the Jira issue link in the squashed commit

### CODE REVIEW

**Review by newswangerd — APPROVED** [review]

File: .compose.env.example [hunk 1]
```diff
@@ -63,3 +63,6 @@ ENABLE_SIGNING=1
 
 ## Enable automated logging to be written to /var/log/galaxy_api_access.log
 # PULP_GALAXY_ENABLE_API_ACCESS_LOG=true
+
+## Customize password lenght validation
```
Comment (ironfroggy) [inline .compose.env.example:67]: typo here with "lenght"

File: docs/config/user_access.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+performed and one can configure the minimum password length.
+
+The default value is 9 and can be customized using:
+
+```python title="/etc/pulp/settings.py"
+GALAXY_MINIMUM_PASSWORD_LENGTH=15
+```
+
+```bash title="envronment variables"
+export PULP_PULP_GALAXY_MINIMUM_PASSWORD_LENGTH=15
```
Comment (ironfroggy) [inline docs/config/user_access.md:20]: Is this correct with the double prefix?

**Review by ironfroggy — CHANGES_REQUESTED** [review]
We will need to file an AAP ticket for the installer to expose this to users.

Does this need any backports?

**Review by jctanner — CHANGES_REQUESTED** [review]
Please add at least one test.

**Review by jctanner — APPROVED** [review]
:+1:

**Review by ironfroggy — APPROVED** [review]

**Pull request merged**